src/acquisition/covidcast/database.py

```diff
@@ -235,21 +239,20 @@ def insert_or_update_batch(self, cc_rows, batch_size=2**20, commit_partial=False
   def get_covidcast_meta(self):
     """Compute and return metadata on all non-WIP COVIDcast signals."""
 
-    meta = []
+    n_threads = 8 # aka number of concurrent db connections, which [sh|c]ould be ~= #cores available to SQL server
```

Comment:
let's do #cores - 1 to be polite

we can get this programmatically with
```
import multiprocessing
n_threads = multiprocessing.cpu_count() - 1
```

Reply:
brilliant!  that could present a small problem if this job runs on different hardware than the db, but we should not run into that issue in prod.